build.cake

```diff
@@ -193,39 +158,21 @@ Task ("NuGet")
 		}
 	});
 
-	// Only build if framework is present
-	if (DirectoryExists ("./PSPDFKit.Mac.Model/PSPDFKit.framework/")) {
-		NuGetPack ("./nuget/pspdfkit-mac-model.nuspec", new NuGetPackSettings {
-			Version = $"{MACVERSION}.{MACOS_SERVICERELEASE_VERSION}+sha.{commit}",
-			OutputDirectory = "./nuget/pkgs/",
-			BasePath = "./",
-			Properties = new Dictionary<string, string> {
-				{"NoWarn", "NU5105"},
-			}
-		});
-	}
 });
 
 Task ("NuGet-Push")
-	.IsDependentOn("Nuget")
```

Comment:
Any reason for this?

Reply:
@dalexsoto The reason is so we don't have to do the whole build again, which takes time, when we just want to push the nugets. The workflow is basically to run `./build.sh --Target=Nuget`, which build the whole thing and creates the nugets. We then test everything and afterwards release the nugets with `./build.sh --Target=Nuget-Push`, since the nugets are already created.